pkg/driver/controller_server.go

```diff
@@ -590,7 +591,32 @@ func (d *Driver) CreateSnapshot(context.Context, *csi.CreateSnapshotRequest) (*c
 }
 
 // DeleteSnapshot is part of implementing Snapshot & Restore functionality, but we don't support that
```

Comment:
@Praveen005 

Thank you for your comment :pray: 
I agree; this should be revised! However, removing it entirely might suggest that this feature is already supported, so I believe it would be better to adjust the phrasing. 
How about the following comment? :thinking: 

```
DeleteSnapshot is part of implementing Snapshot & Restore functionality, and it will be supported in the future.
```

Reply:
Looks good, but will have to be updated later.

something like, "// DeleteSnapshot is implemented to support Snapshot & Restore functionality, allowing snapshots to be deleted as needed." can work too. As this code will be merged only when all pieces are in place I guess. what do you say?

Reply:
Yes, it looks good to me! Thank you :pray: 
> As this code will be merged only when all pieces are in place I guess. what do you say?

I was talking with Ryota about this point. Since this implementation does not particularly affect the existing behavior, we can merge it in this code!